website/static/js/forgotPassword.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * forgot password view model
+ */
+'use srict';
+
+var ko = require('knockout');
+require('knockout-validation');
+
+var $osf = require('osfHelpers');
+
+var ViewModel = function() {
+
+    var self = this;
+
+    self.username = ko.observable('').extend({
+        required: true,
+        email: true
+    });
+
+    self.submit = function() {
```

Comment:
There's some duplication with `signIn.js` here. It might make sense to have a base class whose subclasses define fields and implement `isValid()`.

written in ES6 for brevity (we'd probably use https://github.com/CenterForOpenScience/osf.io/pull/2142 once it is merged):

```js
class ForgotPasswordViewModel extends FormViewModel {
    constructor() {
        this.username = ko.observable('').extend(...);
    }
    isValid() {
        if (!self.username.isValid()) {
            raise ValidationError('Please enter a valid email address.');
        }
    }
};
```

This doesn't have to be implement before this is merged.